generators/server/templates/src/test/java/package/ReactiveSqlTestContainerExtension.java.ejs

```diff
@@ -51,7 +51,6 @@ public class ReactiveSqlTestContainerExtension implements BeforeAllCallback {
             .withTmpFs(Collections.singletonMap("/testtmpfs", "rw"));
     <%_ } else if (prodDatabaseType === 'mssql') { _%>
     private static MSSQLServerContainer<?> container =  new MSSQLServerContainer<>("<%= DOCKER_MSSQL %>")
-            .withDatabaseName("<%= baseName %>")
```

Comment:
Is there a specific reason for this change?

Reply:
Yes, in the MSSQLServerContainer the withDatabaseName is not overriden ( https://github.com/testcontainers/testcontainers-java/blob/1.14.3/modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java ), so the super class's method is called: https://github.com/testcontainers/testcontainers-java/blob/48534069037e17147208e65c3c7d3d03e3c83a47/modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java#L89 - which throws an UnsupportedOperationException().

So it is never used before. Unfortunately, even after my change, there are other issues with the MSSQL support - but at least the tests are not failing in the startup.